test/NuGet.Core.Tests/NuGet.Commands.Test/PackCommandRunnerTests.cs

```diff
@@ -48,6 +48,84 @@ public void RunPackageBuild_WithDefaultExcludes_ExcludesDefaultExcludes()
             }
         }
 
+        [Fact]
+        public void RunPackageBuild_WithInvalidNuspec_Throws()
+        {
+            using (var test = DefaultExclusionsTest.Create(validNuspec: false))
+            {
+                var args = new PackArgs()
+                {
+                    CurrentDirectory = test.CurrentDirectory.FullName,
+                    Exclude = Enumerable.Empty<string>(),
+                    Logger = NullLogger.Instance,
+                    Path = test.NuspecFile.FullName,
+                };
+                var runner = new PackCommandRunner(args, createProjectFactory: null);
+
+                try
+                {
+                    var result = runner.RunPackageBuild();
+                }
+                catch (Exception e)
+                {
+                    Assert.Equal(e.Message, "Version is required.");
```

Comment:
Assert.Throws<>(..) would be better instead of catching to ensure the uncaught exception occurs.